pages.nl/common/base64.md

```diff
@@ -0,0 +1,19 @@
+# base64
+
+> Codeer of decodeer bestand of standaardinvoer van/naar Base64 naar standaarduitvoer.
+
+- Codeer een bestand:
+
+`base64 {{bestandsnaam}}`
+
+- Decodeer een bestand:
+
+`base64 -d {{bestandsnaam}}`
+
+- Codeer van `stdin`:
+
+`{{eencommando}} | base64`
+
+- Decodeer `stdin`:
```

Comment:
I don't speak the language, but `van` is not required here?
```suggestion
- Decodeer van `stdin`:
```

Reply:
In fact, you are correct on that one. I will fix this right away.

Reply:
So `van` is not as translation for `from`?

Reply:
`van` does mean `from`; for the above it is not incorrect, but it is not strictly _required._